checkov/main.py

```diff
@@ -79,8 +79,7 @@
 logger = logging.getLogger(__name__)
 checkov_runners = [value for attr, value in CheckType.__dict__.items() if not attr.startswith("__")]
 
-
-DEFAULT_RUNNERS = (
+DEFAULT_RUNNERS = [
```

Comment:
can you add a comment here, that the `sca_package` runner is added during the run method

Reply:
yeah sure